Documentation/source/Changelog.md

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+4.5.0 (2019-02-28)
+------------------
+
+-   Added flash toggle in the camera screen, configurable view `GiniConfiguration.flashToggleEnabled`.
```

Comment:
configurable view?
or configurable **via**?

Reply:
Via, for some reason _Atom_ corrected it.